### DISCUSSION TITLE
How to  Hide Textareas Border

### DISCUSSION THREAD

**Discussion by ApaulMyLittleAirport** [BlazorComponent/MASA.Blazor#2036] [Q&A] [answered]
Textareas Border can not hide,who tell me how to do ？ thank you varymatch 
<MTextarea Readonly="true"     Outlined Dense RowHeight="0" HideDetails="true" AutoGrow />

**Comment by capdiem** [comment, marked as answer]
Solo Flat

**Reply by ApaulMyLittleAirport** [reply]
yes!! 非常感谢！！！
